index.bs

```diff
@@ -134,29 +126,30 @@ For each <a>element</a>, there is a:
 <div algorithm="additions to update rendering">
     In the <a>update the rendering</a> step of the <a>event loop processing model</a>, add a step right after the step that calls <a>mark paint timing</a>:
 
-    1. Invoke the algorithm to <a>evaluate the layout instability</a> of a document with the complete set of <a>element</a>s for that {{Document}}.
+    1. For each fully active {{Document}} in <em>docs</em>, invoke the algorithm to <a>evaluate the layout instability</a> of that {{Document}}.
 </div>
 
-
 Evaluate Layout Instability {#sec-eval-layout-instability}
 --------------------------------------------------------
 
 <div algorithm="evaluate the layout instability">
-    When asked to <dfn export>evaluate the layout instability</dfn>, with <var>elements</var> as the list of elements to be evaluated, run the following steps:
+    When asked to <dfn export>evaluate the layout instability</dfn> given an active {{Document}} <var>doc</var>, run the following steps:
 
+    1. Let <var>elements</var> be the list of current elements in <var>doc</var>.
```

Comment:
'list of current elements' is not explicit enough. It should be clearer, something like:
`Let |elements| be the set of |document|'s [=descendants=] and [=shadow-including descendants=]`
Actually not sure if shadows are to be included or not, @skobes?

Reply:
sorry it isn't clear to me what descendants are here.  Maybe I am not familiar with the [=*=] notation?

Reply:
I think "set of shadow-including descendants of the document" is what we want.

https://dom.spec.whatwg.org/#concept-shadow-including-descendant

Reply:
done.